pkg/controller/application/executor.go

```diff
@@ -85,7 +86,7 @@ func (r *AppReleaseReconciler) createOrUpgradeAppRelease(ctx context.Context, rl
 		return r.updateStatus(ctx, rls, appv2.StatusFailed, err.Error())
 	}
 
-	return r.updateStatus(ctx, rls, appv2.StatusCreated)
+	return r.updateStatus(ctx, rls, state, "Deploying")
```

Comment:
The front end is not adapted to the translation in this state, and it is recommended to keep it as it is.